src/Payments/Cards/Cards.ts

```diff
@@ -56,11 +56,19 @@ interface CardParameters {
    * The card verification code
    */
   cvc: string
+  /**
+   * Set to true to indicate that this card can be used multiple times
+   */
+  multi_use: boolean
 }
 
 interface CardRecord {
   id: string
   live_mode: boolean
+  multi_use: boolean
+  unavailble_at: string
```

Comment:
@alan: This looks like a typo?

https://duffel.com/docs/api/v2/card/schema#card-schema-unavailable-at